examples/pytorch/GATNE-T/src/main.py

```diff
@@ -0,0 +1,325 @@
+from collections import defaultdict
+import math
+import os
+import sys
+import time
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import tqdm
+from numpy import random
+from torch.nn.parameter import Parameter
+import dgl
+import dgl.function as fn
+
+from utils import *
+
+
+def get_graph(network_data, vocab):
+    ''' Build graph, treat all nodes as the same type
+    
+    Parameters
+    ----------
+    network_data: a dict
+        keys describing the edge types, values representing edges
+    vocab: a dict
+        mapping node IDs to node indices
+
+    Output
+    ------
+    DGLHeteroGraph
+        a heterogenous graph, with one node type and different edge types
+    '''
+    graphs = []
+    num_nodes = len(vocab)
+
+    for edge_type in network_data:
+        tmp_data = network_data[edge_type]
+        edges = []
+        for edge in tmp_data:
+            edges.append((vocab[edge[0]], vocab[edge[1]]))
+            edges.append((vocab[edge[1]], vocab[edge[0]]))
+        g = dgl.graph(edges, 'user', edge_type, num_nodes)
```

Comment:
The graph in GATNE-T only has one node type.  Consider removing the name `user`.

The APIs in turn becomes something like:
```python
block.number_of_dst_nodes('user') -> block.number_of_dst_nodes()
block.srcnodes['user'].data -> block.srcdata
block.dstnodes['user'].data -> block.dstdata
```